WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -67,12 +80,23 @@ private set
 		public EndPoint? TorSocks5EndPoint { get; private set; }
 		public bool IsTorUsed => TorSocks5EndPoint is { };
 
-		public bool IsolateStream { get; private set; }
+		private bool IsolateStream { get; }
 
-		public TorSocks5Client TorSocks5Client { get; private set; }
+		private TorSocks5Client? TorSocks5Client { get; set; }
 
 		private static AsyncLock AsyncLock { get; } = new AsyncLock(); // We make everything synchronous, so slow, but at least stable.
 
+		private static async Task<HttpResponseMessage> ClearnetRequestAsync(HttpRequestMessage request)
+		{
+			request.Version = HttpProtocol.HTTP11.Version;
+			request.Headers.AcceptCharset.Add(new StringWithQualityHeaderValue("utf-8"));
+
+			//request.Headers.TransferEncodingChunked = true;
```

Comment:
Should this get removed?

Reply:
I have refactored the code more. It's bigger change but it's nicer.